app/file_service.py

```diff
@@ -0,0 +1,46 @@
+import random
+
+from fastapi import UploadFile
+
+from app.custom_expetion import ImageException
+import os
+
+allowed_MIME = ["image/jpeg", "image/png"]
+max_image_size_KB = 500
+
+# TODO(any): implemant save file in cloud service
+
+def save_file(data: bytes, extension: str):
+    name = get_random_string(10) + "." + extension
+    file = open(name, "wb")
+    file.write(data)
+    file.close()
+    return name
+
+
+def get_random_string(length):
```

Comment:
what if this random string will be the same as the name of an already existing file?